lib/rootApi.js

```diff
@@ -14,6 +14,9 @@ const {validate} = require('bedrock-validation');
 const validator = require('./validator');
 const LedgerNode = require('./LedgerNode');
 
+const {default: {LruMemoize}} = require('@digitalbazaar/lru-memoize');
+const lruMemoize = new LruMemoize({max: 1000});
```

Comment:
Is there a default `maxAge`? It seems that we are asserting that it's safe to cache the ledgerNode document indefinitely. If that is the case, can we capture that in a comment that this is intended behavior.

Reply:
another good catch, on second thought, it's not safe, added a 5 min `maxAge`.